pandas/tests/plotting/test_series.py

```diff
@@ -684,11 +684,13 @@ def test_kind_both_ways(self):
         kinds = (
             plotting.PlotAccessor._common_kinds + plotting.PlotAccessor._series_kinds
         )
-        _, ax = self.plt.subplots()
         for kind in kinds:
-
+            _, ax = self.plt.subplots()
```

Comment:
Some strange interaction / issue when plotting on existing axes. IIRC it's from MPL being stricter about things. I didn't investigate too much.

Reply:
Its preferable to keep the reference to the figure and axes and then reuse, rather than depend on the implicit ability of `plt.subplots()` to pick up the same axes...